src/vulkan/vk-device.cpp

```diff
@@ -1424,7 +1424,17 @@ Result DeviceImpl::getTextureAllocationInfo(const TextureDesc& desc_, Size* outS
 
 Result DeviceImpl::getTextureRowAlignment(Format format, Size* outAlignment)
 {
-    *outAlignment = 1;
+    switch (format)
```

Comment:
Did you check if there is any Vulkan specific way to get to the row alignment requirements?